experimental/PropertyDDS/examples/property_inspector/package.json

```diff
@@ -59,6 +64,18 @@
     "sass-loader": "^7.1.0",
     "style-loader": "^1.0.0",
     "node-sass": "^4.9.3",
-    "source-map-loader": "^0.2.4"
-  }
+    "source-map-loader": "^0.2.4",
+    "jest": "^26.6.3",
+    "jest-junit": "^10.0.0",
+    "jest-puppeteer": "^4.3.0",
+    "puppeteer": "^1.20.0"
+  },
+  "jest-junit": {
+    "outputDirectory": "nyc",
+    "outputName": "jest-junit-report.xml"
+  },
+  "test": "npm run test:jest",
+  "test:jest": "jest",
+  "test:jest:verbose": "cross-env FLUID_TEST_VERBOSE=1 jest"
```

Comment:
is this redundant? they are already in the scripts section higher up

Reply:
yes, copied accidentally. Its redundant indeed.